eduNEXT/eox-nelp
# Jlc/update permission for data download tab

## Description

This adds a permission to view the tab "data download" for instructors and staff of course.

## Testing instructions
Checkout this branch.
Login and create a normal user.
Then give it `staff` or `instructor` role.
![Screenshot from 2024-02-14 19-16-44](https://github.com/eduNEXT/eox-nelp/assets/51926076/c1282d98-5bf1-4b8b-93bb-2cf2a421043c)

Check with that user you can view the desired tab.
### Before
![before_instructor](https://github.com/eduNEXT/eox-nelp/assets/51926076/1c7acb31-f406-4d57-8285-901107a1525a)

### After
![after_instructor](https://github.com/eduNEXT/eox-nelp/assets/51926076/414a6b99-bec6-493a-9808-931ebba01073)

## Additional information

Jira story

## Checklist for Merge
